### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+##### ISSUE TYPE
+
+This issue tracker is only for the bug reports, for different issues see:
+
+ - Feature Idea: Please report feature ideas in the
+   [RFC repo](https://github.com/LibrePCB/librepcb-rfcs)
+ - Documentation Report: Please report documentation issues in the
+   [documentation repo](https://github.com/LibrePCB/librepcb-doc)
+ - Bug Report: You have come to the right place!
```
Comment (rnestler) [inline .github/ISSUE_TEMPLATE.md]: I'm not sure if I should place this whole section in a HTML comment, so it won't show up in the actual bug report. On the other hand this makes it more visible.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!---
+Verify first that your issue/request is not already reported on GitHub.
```
Comment (ubruhin) [inline .github/ISSUE_TEMPLATE.md]: Maybe start this sentence with "Please..."? :)

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+This issue tracker is only for the bug reports, for different issues see:
```
Comment (dbrgn) [inline .github/ISSUE_TEMPLATE.md]: You can remove "the". Also, I'd replace "different" with "other".

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ - Bug Report: You have come to the right place! ☺
+-->
+
+##### LibrePCB VERSION
+<!--- Paste the git hash of the LibrePCB version used below. You can find the
+version in the first line of the terminal output or in the "Help->About"
+dialog. -->
+
+##### OS / ENVIRONMENT
+<!--- Mention the OS you are running LibrePCB from. -->
```
Comment (dbrgn) [inline .github/ISSUE_TEMPLATE.md]: s/from/on/

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (rnestler) [reply .github/ISSUE_TEMPLATE.md]: Done.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (rnestler) [reply .github/ISSUE_TEMPLATE.md]: Done.

**Pull request merged**